### PR TITLE
Space pod construction fixes

### LINKED ISSUE #25736: No mainboard/core needed for Spacepod construction
(WEB REPORT BY: badgoyguy REMOTE: 206.221.180.138:7777)
# Revision
31af501e0305da9379a7ace22ca772c7cd80411b

# Description
you can skip the Space pod core/mainboard and have a functioning spacepod.
# Steps to Reproduce
build a spacepod

# What you Expected
steps 8-14 at http://ss13.moe/wiki/index.php/Space_Pod

# What Actually Happened
steps 8-14 are skipable and you cant even put the things in. Just weld the armor on and you're good to go

### LINKED ISSUE #27506: You can skip the pod construction
(WEB REPORT BY: casualbro REMOTE: 206.221.180.138:7777)
1. Build the frame parts
2. Attach frames
3. Immediately use armor, which will skip you to the last 2 stages without having to attach the core or anything
4. Free Pod!

### PR DESCRIPTION
**Pull request by Zth--**
Closes #27506
Closes #25736

Not sure if #25873 is the same or something different is going on. Could not reproduce with those steps

### CODE REVIEW

**Comment by anunknownentity03** [comment]
FUUUUUCK someone told the coders

**Comment by dunsel-hordon** [comment]
STAAAAP DO NOT MERGE EVERYTHING IS WORKING AS INTENDED NO BUG FIX NEEDED!

File: code/modules/spacepods/spacepods.dm [hunk 1]
```diff
@@ -757,6 +760,60 @@
...
+	actions_types = list(/datum/action/spacepod/pilot/toggle_lights) // I'm okay with a 4-tile lantern
+	use_internal_tank = FALSE
+
+/obj/spacepod/empty/attackby(obj/item/W, mob/user)
+	if(iscrowbar(W))
+		hatch_open = !hatch_open
+		to_chat(user, "<span class='notice'>You [hatch_open ? "open" : "close"] the maintenance hatch. [hatch_open ? "There's nothing inside." : ""]</span>")
+		if(hatch_open && contents.len)
+			var/anyitem = 0
+			if(anyitem)
```
Comment (DamianX) [inline code/modules/spacepods/spacepods.dm]: this is always false

Comment (Zth--) [reply code/modules/spacepods/spacepods.dm]: leftover I forgot to remove

**Comment by DrSnips** [comment]
POD BABIES POD BABIES POD BABIES

File: code/modules/spacepods/spacepods.dm [hunk 2]
```diff
@@ -757,6 +760,56 @@
...
+		else
+			to_chat(user, "<span class='notice'>You stare at the hatch. There's no circuits here.</span>")
+			return
+	if(W.force)
+		visible_message("<span class = 'warning'>\The [user] hits \the [src] with \the [W]</span>")
+		adjust_health(W.force)
+		W.on_attack(src, user)
+
+/obj/spacepod/empty/attack_hand(mob/user)
+	user.visible_message("<span class='notice'>[user] pets [src]!</span>", \
```
Comment (Kurfursten) [inline code/modules/spacepods/spacepods.dm:788]: This baby can hold so many air canisters

**Pull request merged**